File: pset6/Mario/mario.py
from cs50 import get_int


def printChar(char):
    print(char, end='')


height = 0
while height < 1 or height > 8:
    height = get_int("Height: ")

for i in range(height):
    for j in range(height - (i + 1)):
        printChar(" ")

    for j in range(i + 1):
        printChar("#")

    for j in range(2):
        printChar(" ")

    for j in range(i + 1):
        printChar("#")

    print()

File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

//Initializing Function
void printStr(int n, string digit);

int main(void)
{
    int size;
    do 
    {
        //Prompts the user the height of the pyramid
        size = get_int("Height: ");
    }
    while (size < 1 || size > 8);

    for (int i = 1; i <= size; i++)
    {
        printStr(size - i, " "); // Whitespace to the left of the pound
        printStr(i, "#");
        printStr(2, " "); // Space between the pyramid
        printStr(i, "#");
        printf("\n"); // New Line
    }
}
/*
* param(n) == Number of times to print a given string
* param(digit) == String to be print
*/
void printStr(int n, string digit)
{
    for (int i = 0; i < n; i++)
    {
        printf("%s", digit);
    }
};